front/assets/style/base.scss

```diff
@@ -102,11 +107,20 @@
 
 /* dark theme styles */
 .theme--dark.v-btn {
-  background: $primary-color !important;
-  color: white !important;
+  background-color: $btn-default-background !important;
```

Comment:
color should be specified on per button basis via color prop if we do this way we might get troubles overriding it